ambuda/tasks/ocr.py

```diff
@@ -0,0 +1,102 @@
+"""Background tasks for proofing projects."""
+
+import time
+from typing import Optional
+
+from celery import group
+from celery.result import GroupResult
+
+from ambuda import consts
+from ambuda import database as db
+from ambuda.enums import SitePageStatus
+from ambuda.tasks import app
+from ambuda.tasks.utils import TaskStatus, CeleryTaskStatus
+from ambuda.utils import google_ocr
+from ambuda.utils.assets import get_page_image_filepath
+from ambuda.utils.revisions import add_revision
+from ambuda import queries as q
+from config import create_config_only_app
+
+
+def _run_ocr_for_page_inner(
+    app_env: str,
+    project_slug: str,
+    page_slug: str,
+) -> int:
+    """Must run in the application context."""
+
+    flask_app = create_config_only_app(app_env)
+    with flask_app.app_context():
+        bot_user = q.user(consts.BOT_USERNAME)
+        if bot_user is None:
+            raise ValueError(f'User "{bot_user}" is not defined.')
+
+        # The actual API call.
+        image_path = get_page_image_filepath(project_slug, page_slug)
+        content = google_ocr.full_text_annotation(image_path)
```

Comment:
If I understand correctly, this is just the text. I feel very strongly that it would be good to save the actual API response somewhere — having the bounding boxes will come in handy for the line-by-line view (and other future improvements we can think of).

Reply:
See #319.